server/service/handler.go

```diff
@@ -637,3 +650,124 @@ func RedirectSetupToLogin(svc fleet.Service, logger kitlog.Logger, next http.Han
 		next.ServeHTTP(w, r)
 	}
 }
+
```

Comment:
all of the code below is not new, it's the code I moved from the now deleted `cmd/fleet/apple_mdm.go`